hie-plugin-api/Haskell/Ide/Engine/ModuleCache.hs

```diff
@@ -117,68 +115,52 @@ loadCradle iniDynFlags (NewCradle fp) = do
   traceShowM cradle
   liftIO (GHC.newHscEnv iniDynFlags) >>= GHC.setSession
   liftIO $ setCurrentDirectory (BIOS.cradleRootDir cradle)
-  res <- gcatches
-    (do
-      withProgress "Initialising Cradle" NotCancellable (initializeCradle cradle)
-      return $ IdeResultOk ()
-    )
-    [ ErrorHandler $
-      \(err :: GHC.GhcException) -> do
+  res <- withProgress "Initialising Cradle" NotCancellable (\f ->
+          BIOS.initializeFlagsWithCradleWithMessage (Just (toMessager f)) fp (fixCradle cradle)
+         )
+  case res of
+    BIOS.CradleNone -> return (IdeResultOk ())
```

Comment:
Should we cache files in the none-cradle?

Reply:
I think we need to make `h-i-e` understand the `None` cradle properly. It will still try to load files with this change.

Reply:
What does understand mean? That it should not even try to load, or would it be fine, if it tries to load the file, it immediately succeeds with no results?
Since it is an additional feature, we dont have to get it completely right from the beginning, in my opinion.